packages/sol-tracing-utils/src/artifact_adapters/sol_compiler_artifact_adapter.ts

```diff
@@ -40,14 +47,15 @@ export class SolCompilerArtifactAdapter extends AbstractArtifactAdapter {
         this._resolver.appendResolver(new NPMResolver(packagePath));
         this._resolver.appendResolver(new RelativeFSResolver(this._sourcesPath));
         this._resolver.appendResolver(new FSResolver());
+        this._resolver.appendResolver(new NameResolver(this._sourcesPath));
```

Comment:
I guess at some point our artifacts started using relative imports in `sources`? anyway NameResolver solves this issue